spec/builtins.test.sh

```diff
@@ -54,6 +54,47 @@ cd-nonexistent
 status=0
 ## END
 
+### cd permits double bare dash
+cd -- /
+echo $PWD
+# stdout: /
+
+### cd to non-symlink with `-P`
```

Comment:
same nitpick about backticks.

Also I think it's safe to say that $TMP is never going to have spaces, since it's controlled by `test/sh_spec.py`.  I tend to leave off quotes but I know that most people are in the habit of using quotes.  That is one of the things that Oil is supposed to fix :)  (i.e. not needing quotes everywhere "just in case")

Reply:
Ah, I didn't realize that `$TMP` wouldn't be inherited, if available, from the parent environment. I was just reading [Safe ways to do things in Bash](https://github.com/anordal/shellharden/blob/master/how_to_do_things_safely_in_bash.md), which suggests using quotes even when not necessary. I'll change it to be consistent with the existing tests, though.